samples/source/concurrent-dispatch/200-perf-test.yaml

```diff
@@ -0,0 +1,78 @@
+# RabbitMQ cluster used by the Source
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: rabbitmq
+  namespace: concurrent-dispatch
+spec:
+  replicas: 1
+---
```

Comment:
This should be split into a different YAML file. `200-rabbitmq.yaml` seems appropriate.